tests/core/resource/test_viewset.py

```diff
@@ -53,43 +55,36 @@ def test_arguments_are_merged_on_initialization(self):
     def test_default_arguments_are_copied_before_being_returned(self):
         original_arguments = {}
         viewset = ViewSet(
-            collection_get_arguments=original_arguments,
-            validate_schema_for=()
-        )
-        arguments = viewset.collection_arguments(mock.sentinel.resource, 'GET')
+            collection_get_arguments=original_arguments)
+        arguments = viewset.collection_arguments(mock.MagicMock(), 'GET')
         self.assertEquals(original_arguments, {})
         self.assertNotEquals(original_arguments, arguments)
 
     def test_permission_private_is_set_by_default(self):
         viewset = ViewSet()
-        args = viewset.collection_arguments(mock.sentinel.resource, 'GET')
+        args = viewset.collection_arguments(mock.MagicMock(), 'GET')
         self.assertEquals(args['permission'], 'private')
 
     def test_schema_is_added_when_method_matches(self):
-        viewset = ViewSet(
-            validate_schema_for=('GET', )
-        )
+        viewset = ViewSet()
```

Comment:
Yes, we are. It was used to tell if we added a body to the request schema or not, but it doesn't make sense anymore as we split the requests into `RequestSchema` and `PayloadRequestSchema`.

We could consider this a breaking change because it can be changed from a resource, but I can't see any reason why someone would change this attribute, because probably the requests would fail if it was different.